### DISCUSSION TITLE
Crouton

### DISCUSSION THREAD

**Discussion by lorenkjr1025** [community/community#53048] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

I am having a problem installing crouton on my chromebook.  It does not see to be a hardware related problem, as it does not work with kali,debian,or ubuntu.  I sent you a copy of part of the crouton install, I hope it is enough.  It says it can not read cras messages.  This been for months with out any relief, and I would like to found out how to deal with it.  Thank for taking the time to read this.
Setting up lib32gcc-11-dev (11.3.0-1ubuntu1~22.04) ...
Setting up gcc (4:11.2.0-1ubuntu1) ...
Setting up pkg-config (0.29.2-1ubuntu3) ...
Setting up libnsl-dev:i386 (1.3.0-2build2) ...
Setting up libc6-dev:i386 (2.35-0ubuntu3.1) ...
Setting up gcc-11-multilib (11.3.0-1ubuntu1~22.04) ...
Setting up gcc-multilib (4:11.2.0-1ubuntu1) ...
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...
gcc set to automatically installed.
gcc-multilib set to automatically installed.
libc6-dev:i386 set to automatically installed.
pkg-config set to automatically installed.
libspeexdsp-dev:i386 set to automatically installed.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  cpp cpp-11 gcc gcc-11 gcc-11-base gcc-11-multilib gcc-multilib lib32asan6 lib32atomic1 lib32gcc-11-dev lib32gcc-s1 lib32gomp1
  lib32itm1 lib32quadmath0 lib32stdc++6 lib32ubsan1 libasan6 libatomic1 libc-dev-bin libc6-dev libc6-dev:i386 libc6-dev-i386
  libc6-dev-x32 libc6-i386 libc6-x32 libcc1-0 libcrypt-dev libcrypt-dev:i386 libdpkg-perl libgcc-11-dev libisl23 libitm1 liblsan0
  libmpc3 libmpfr6 libnsl-dev libnsl-dev:i386 libquadmath0 libspeex-dev:i386 libspeex1:i386 libspeexdsp-dev:i386 libtirpc-dev
  libtirpc-dev:i386 libtsan0 libubsan1 libx32asan6 libx32atomic1 libx32gcc-11-dev libx32gcc-s1 libx32gomp1 libx32itm1 libx32quadmath0
  libx32stdc++6 libx32ubsan1 linux-libc-dev linux-libc-dev:i386 pkg-config rpcsvc-proto
Use 'apt autoremove' to remove them.
Suggested packages:
  libasound2-doc:i386
The following NEW packages will be installed:
  libasound2-dev:i386
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 110 kB of archives.
After this operation, 693 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu jammy/main i386 libasound2-dev i386 1.2.6.1-1ubuntu1 [110 kB]
Fetched 110 kB in 1s (84.5 kB/s)              
Selecting previously unselected package libasound2-dev:i386.
(Reading database ... 19946 files and directories currently installed.)
Preparing to unpack .../libasound2-dev_1.2.6.1-1ubuntu1_i386.deb ...
Unpacking libasound2-dev:i386 (1.2.6.1-1ubuntu1) ...
Setting up libasound2-dev:i386 (1.2.6.1-1ubuntu1) ...
libasound2-dev:i386 set to automatically installed.
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following packages were automatically installed and are no longer required:
  cpp cpp-11 gcc gcc-11 gcc-11-base gcc-11-multilib gcc-multilib lib32asan6 lib32atomic1 lib32gcc-11-dev lib32gcc-s1 lib32gomp1
  lib32itm1 lib32quadmath0 lib32stdc++6 lib32ubsan1 libasan6 libasound2-dev:i386 libatomic1 libc-dev-bin libc6-dev libc6-dev:i386
  libc6-dev-i386 libc6-dev-x32 libc6-i386 libc6-x32 libcc1-0 libcrypt-dev libcrypt-dev:i386 libdpkg-perl libgcc-11-dev libisl23 libitm1
  liblsan0 libmpc3 libmpfr6 libnsl-dev libnsl-dev:i386 libquadmath0 libspeex-dev:i386 libspeex1:i386 libspeexdsp-dev:i386 libtirpc-dev
  libtirpc-dev:i386 libtsan0 libubsan1 libx32asan6 libx32atomic1 libx32gcc-11-dev libx32gcc-s1 libx32gomp1 libx32itm1 libx32quadmath0
  libx32stdc++6 libx32ubsan1 linux-libc-dev linux-libc-dev:i386 pkg-config rpcsvc-proto
Use 'apt autoremove' to remove them.
Suggested packages:
  ed diffutils-doc
The following NEW packages will be installed:
  patch
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 109 kB of archives.
After this operation, 234 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu jammy/main amd64 patch amd64 2.7.6-7build2 [109 kB]
Fetched 109 kB in 2s (52.9 kB/s)
Selecting previously unselected package patch.
(Reading database ... 20025 files and directories currently installed.)
Preparing to unpack .../patch_2.7.6-7build2_amd64.deb ...
Unpacking patch (2.7.6-7build2) ...
Setting up patch (2.7.6-7build2) ...
patch set to automatically installed.
sed: can't read common/cras_messages.h: No such file or directory
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/jammy...
chronos@localhost / $  / $ 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

Also please note: You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
